crates/dojo-types/src/event.rs

```diff
@@ -1,5 +1,7 @@
 use starknet::core::types::Felt;
 
+pub const SYSTEM_EVENT_SELECTOR: &str = "SystemEvent__";
```

Comment:
Just curious as to why we have that for the event messages? Can't we just do something similar to StoreSetRecord? Like EventMessage and define a wrapper struct?

Reply:
Hum, we could do something like:
```rust
pub struct EventMessage {
    #[keys]
    pub keys: Span<felt252>,
    pub values: Span<felt252>,
}
```
The problem with that is in the keys we will have the `span` length. But using a manual selector, we're not injecting the length of the `span` into the keys, which may break the indexing and query by keys.